torch/_inductor/ir.py

```diff
@@ -3349,6 +3349,53 @@ def get_template_tiling(self):
         )
 
 
+# Port from aten/src/ATen/native/ConvUtils.h: _conv_input_size
+def _conv_input_size(
+    output_size, weight_size, padding, output_padding, stride, dilation, groups
+):
```

Comment:
How about we keep this function local to `_prepare_convolution_fusion_create` first? It is only used by this function.

Reply:
Made this function local to `_prepare_convolution_fusion_create`.